packages/collector/src/agentConnection.js

```diff
@@ -384,7 +384,7 @@ exports.sendTracingMetricsToAgent = function sendTracingMetricsToAgent(tracingMe
  * @param {boolean} [ignore404]
  * @returns
  */
-function sendData(path, data, cb, ignore404 = false) {
+function sendData(path, data, cb, ignore404 = false, ignorePayloadSizeCheck = false) {
```

Comment:
Its too much effort to refactor the function arguments. I continued with adding the parameter after the callback.
We should refactor that at one point.